src/App.tsx

```diff
@@ -1,51 +1,64 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
+
 import './App.scss';
+import { SortType } from './enums/sort-type.enum';
+import { Good } from './types/Good.type';
+import { goodsFromServer } from './data/goods-from-server';
+import { GoodList } from './components/GoodList';
+import { ButtonGroup } from './components/ButtonGroup';
+
+function getPreparedGoods(
+  goods: Good[],
+  sortField: SortType | '',
+  isReversed: boolean,
+): Good[] {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.ALPHABETICALLY:
+          return good1.localeCompare(good2);
```

Comment:
The `localeCompare` method is used to compare two strings in locale-specific order. However, `good1` and `good2` are of type `Good`, which is an object, not a string. You need to specify the property of `Good` that you want to compare alphabetically. This is a [CODE KNOWLEDGE] issue.

Reply:
type Good is string.